.markdownlint.json

```diff
@@ -99,11 +99,11 @@
   // MD044/proper-names - Proper names should have the correct capitalization
   "MD044": {
     "names": [
+      "ESLint",
       "JavaScript",
-      "TypeScript",
       "TSLint",
-      "ESLint",
-      "typescript-eslint"
+      "typescript-eslint",
+      "TypeScript"
```

Comment:
No actual changes here, just sorting. It happens automatically sometimes anyway.